README.md

```diff
@@ -65,6 +65,8 @@ children = [
 Supervisor.start_link(children, ...)
 ```
 
+To use [Datadog's distributions metric type](https://docs.datadoghq.com/metrics/distributions/), add `formatter_options: [report_as: :distribution]` to your metric.
```

Comment:
Would you mind moving this line to the section describing the Distribution metric?

Reply:
What section do you mean?

Reply:
Somewhere below here 👍  https://github.com/beam-telemetry/telemetry_metrics_statsd/blob/e355ef748520ae7e560dce73c589ab92c9b334fc/lib/telemetry_metrics_statsd.ex#L161-L162